src/main/java/uk/nhs/hee/tis/trainee/sync/service/PostSyncService.java

```diff
@@ -33,10 +37,16 @@
 @Service("tcs-Post")
 public class PostSyncService implements SyncService {
 
+  private static final Logger LOG = LoggerFactory.getLogger(PlacementSyncService.class);
+
```

Comment:
Can use the `@Slf4j` annotation to create a logger for use named `log`, should be in the latest code from master.